src/ngspicetoModelica/ModelicaUI.py

```diff
@@ -52,7 +52,7 @@ def browseFile(self):
     def callConverter(self):
 
         try:
-            self.cmd1 = "python ../ngspicetoModelica/NgspicetoModelica.py " + \
+            self.cmd1 = "python3 ../ngspicetoModelica/NgspicetoModelica.py" + \
```

Comment:
```suggestion
            self.cmd1 = "python3 ../ngspicetoModelica/NgspicetoModelica.py " + \
```
space missing after .py